### ISSUE TITLE
Allow Others on Home Network

### ISSUE THREAD

**Issue by fazi13** [reverie89/gtav-portblock#1]
How do I whitelist other people on my home network using the whitelist file?
I added my own public IP to the whitelist file but other people in my home cannot connect

**Comment by reverie89** [comment]
Try whitelisting the LAN IPs that you want to allow connecting instead

**Comment by fazi13** [comment]
I did some wireshark logging
I think this is why it no longer works:
Rockstar patched it and is no longer sending the packet directly from your friend's IP
They must be sending the data to Rockstar, then Rockstar is sending an encrypted packet to you with the details
if this step fails (which it does due to blocking Rockstar ports) your friends either disconnect or can never connect to your session

so basically this only works now to allow you to play solo, can't play with friends
